src/views/ipv/page/page-multiple-doc-check.njk

```diff
@@ -19,6 +19,7 @@
     {% endfor %}
   </ul>
   <p class="govuk-body">{{ 'pages.pageMultipleDocCheck.content.paragraph2' | translate | safe }}</p>
+  <p class="govuk-body">{{ 'pages.pageMultipleDocCheck.content.paragraph3' | translate | safe }}</p>
```

Comment:
Do we need the `safe` filter here? Feels like we've not got any content that would get escaped?

Reply:
In the welsh translation there is a ' which might need it?

Reply:
I think that's just an apostrophe? And actually looking at it closer I think it should be a `’` rather than a `'`

Reply:
Yeah we don't need it for Welsh characters, it's only necessary if we are inputting html via the translation string

Reply:
Good eyes, I'll remove the safe, the `'` is in a number of places in welsh translations (I copied it from another place) so might be worth another ticket to correct them?

Reply:
Yep, happy to punt the apostrophe to another ticket